guides/v2.2/coding-standards/technical-guidelines.md

```diff
@@ -612,6 +621,8 @@ class View extends Template
 
 6.4.4.13. A service contract MUST NOT rely on the execution context (application area). The service implementation MUST NOT depend on the application state.
 
+6.4.4.14. A service contract SHOULD be an [idempotent method](https://tools.ietf.org/html/rfc7231#section-4.2.2).
```

Comment:
Are we talking about POST (create) methods too?

This item sounds weird, @keharper please check if it needs to be rephrased.

Reply:
Add same statement for cron.

Reply:
I think that is fine.